src/main/java/com/padmaja/inheritance/Test.java

```diff
@@ -0,0 +1,81 @@
+package com.padmaja.inheritance;//Java program to illustrate the
+// concept of inheritance 
+
+// base class 
+class Bicycle 
```

Comment:
why are you putting 3 classes in same file.
Make sure the code is easily understandable.
Please make sure you put classes in different files please.
Class Bicycle, Class MountainBike & class Test